db_scripts/README.md

```diff
@@ -0,0 +1,32 @@
+# e-Signet Database
```

Comment:
eSignet and Identity provider references from the readme file to be removed

Reply:
removed the references

Reply:
"Open ID based Identity provider for large scale authentication." - this is still there

Reply:
removed